t/URT/t/52_limit_cache_size.t

```diff
@@ -156,7 +161,7 @@ sub count_things_in_cache {
 #        foreach (values %{$UR::Context::all_objects_loaded->{$c}} ) {
 #            print "\tid ",$_->id,"\n";
 #        }
-        $count += scalar(values %{$UR::Context::all_objects_loaded->{$c}});
+        $count += scalar(grep { defined } values %{$UR::Context::all_objects_loaded->{$c}});
```

Comment:
No, that would alter the counts for every other assertion.

Besides, the pruner keeps track of the count of actual objects, not ID keys in the cache.